_posts/2018-01-25-fair.md

```diff
@@ -62,4 +62,4 @@ data AIR and look forward to fully achieving the goals of the FAIR principles.
 [^1]: [https://www.force11.org/group/fairgroup/fairprinciples](https://www.force11.org/group/fairgroup/fairprinciples)
 [^2]: [Williams et al, 2017](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536224/)
 [^3]: [Schleicher et al, 2017](http://dx.doi.org/10.1098/rsob.170099)
-[^4]: [Complete workflow for hosting data for a publication](https://docs.openmicroscopy.org/latest/omero/sysadmins/publication.html)
+[^4]: [Complete workflow for hosting data for a publication](https://docs.openmicroscopy.org/latest/omero/sysadmins/public.html)
```

Comment:
Not sure if we've done it elsewhere, but we can add an anchor to the given section `public.html#hosting_data_example` if desired. (Alternative names also welcome on https://github.com/openmicroscopy/ome-documentation/pull/1830)

Reply:
That I am leery of doing given we don't have a linkchecker on this - we mess with anchor links much more often than URLs.

Reply:
And don't redirect them